.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# CircleCI 2.0 configuration file using base image
+# Check https://circleci.com/docs/2.0/circleci-images/#circleci-base-image for more details
+version: 2.1
+jobs:
+  deploy:
+    docker:
+      - image: cimg/base:2020.01
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install CF CLI
+          command: |
+            mkdir -p $HOME/bin
+            export PATH=$HOME/bin:$PATH
+            curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&version=7.1.0" | tar xzv -C $HOME/bin
+            cf install-plugin autopilot -f -r CF-Community
+
+      - deploy:
+          name: Deploy Proxy
+          command: |
+            export PATH=$HOME/bin:$PATH
+            ./bin/cf_deploy.sh proxy fec-beta-fec $CIRCLE_BRANCH
+
+workflows:
+  version: 2.1
+  manual-appoval-and-deploy:
+    jobs:
+      - manual-approval:
+          type: approval
+      # Deploy won't run without manual approval
+      # See https://circleci.com/docs/2.0/workflows/#holding-a-workflow-for-a-manual-approval
+      - deploy:
+          requires:
+            - manual-approval
```

Comment:
I think only approval on the PR is good enough, that's how we do it with our other apps.

Reply:
👍 thanks @patphongs! Modified with 8e4953a